examples/compressed_workflow.yaml

```diff
@@ -0,0 +1,26 @@
+# Stress test to test upper bounds of concurrent pods
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: pod-limits-
```

Comment:
Please make the generate name the name of the file. It makes it difficult to debug during e2e testing which workflow failed. Also, since this is a test and not an example, it probably belongs under test/e2e/functional

Reply:
Sorry these files by mistake got it in PR I will revert it